Aerolyzer/app/views.py

```diff
@@ -59,16 +62,87 @@ def gallery(request):
 
 @login_required
 def upload(request):
-    return render(request,
-    'app/upload.html',
-	{ 'user': request.user },
-    )
+	if request.method == 'POST' and request.FILES['myfile']:
+		myfile = request.FILES['myfile']
+		fs = FileSystemStorage()
+		filename = fs.save(myfile.name, myfile)
+		uploadedFileUrl = fs.url(filename)
+		request.session['filename'] = filename
+		request.session['uploadedFileUrl'] = uploadedFileUrl
+		# verified = imgRestFuncs(filename)
```

Comment:
Here the idea is that the imgRestFuncs would take one argument (the filename in this case but could be changed to the location of the file locally, etc.) and return a tuple: a bool value to determine whether the verification passed and the associated exif data (if the bool value is True) or the specific verification that failed to be used as an error message (if the bool value is False)